tests/integration/targets/api/tasks/main.yml

```diff
@@ -3,6 +3,7 @@
     SC_HOST: "{{ sc_host }}"
     SC_USERNAME: "{{ sc_username }}"
     SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout|default(None) or 10 }}"
```

Comment:
Only default wasn't working here.
Timeout can be empty string instead of None and is treated as a string.

Reply:
I think "{{ sc_timeout|default(10) }}" (or "{{ sc_timeout|default('10') }}" ) would also work.

Reply:
I tried that first, but if `sc_timeout` can be `' '` instead of `None `- which is treated as a `string `and not `float`.
I was getting `"cannot convert string to float exception"`.

Reply:
maybe problem is that we have no `secrets.SC_TIMEOUT` in github.